### ISSUE TITLE
Incomplete type for scrollItem parameter

### ISSUE THREAD

**Issue by dricholm** [hipstersmoothie/react-glider#45]
Hello, first of all thank you for this package!

I have a minor issue with typings in the plugin. `scrollItem()` expects `number` as first parameter in the typing as declared in the package:

```ts
scrollItem(slideIndex: number, isActuallyDotIndex?: boolean): void;
```

However the official [glider-js documentation](https://nickpiscitelli.github.io/Glider.js/) states it can have the following arguments: "Arguments: String | Number, Boolean". So having a string as first argument is also valid. If `prev` is provided it will slide to the previous slide, otherwise to the next slide it seems.
I currently use it with both `prev` and `next` with TypeScript ignore comment because of the package typings. The reason for this use case is because it is used in an event listener on arrow key presses.

Can the parameter for `scrollItem()` be updated to reflect this?

**Comment by hipstersmoothie** [comment]
Make a PR and I'll merge it asap!

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.1.0`](https://github.com/hipstersmoothie/react-glider/releases/tag/v2.1.0) :rocket:
